media-libs/jasper/jasper-2.0.33.recipe

```diff
@@ -0,0 +1,105 @@
+SUMMARY="Implementation of the codec specified in the JPEG-2000 Part-1 standard"
+DESCRIPTION="The JasPer Project is an open-source initiative to provide a \
+free software-based reference implementation of the codec specified in the \
+JPEG-2000 Part-1 standard. More details about this software can be found in \
+the JasPer Software Reference Manual."
+HOMEPAGE="http://www.ece.uvic.ca/~frodo/jasper/"
+COPYRIGHT="2001-2021 Michael D. Adams"
+LICENSE="JasPer v2"
+REVISION="1"
+SOURCE_URI="https://github.com/mdadams/jasper/archive/version-$portVersion.tar.gz"
+CHECKSUM_SHA256="38b8f74565ee9e7fec44657e69adb5c9b2a966ca5947ced5717cde18a7d2eca6"
+SOURCE_DIR="jasper-version-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
```

Comment:
I don't think gcc2 can be removed here. This is used by Haiku's image translators.

Reply:
We are still using the old 2.0.16 recipe for x86_gcc2/x86 for now.

Reply:
@jmairboeck I think the translators use the gcc2 version right?

Reply:
Yes, I think translators are available for both architectures on x86, but core components like Tracker (for thumbnails) and ShowImage definitely use gcc2.